### PR TITLE
#30674

### PR DESCRIPTION
**Pull request by Igloczek**
https://dev.snowdog.pro/issues/30674

### CODE REVIEW

File: components/02-elements/list/_list.scss [hunk 1]
```diff
@@ -43,6 +46,31 @@ $list-divider: 1px solid $gray-dark !default;
...
+                border-bottom: none;
+            }
+            @include mq($screen-m) {
+                border-bottom: 0;
+            }
+        }
+        .list__link {
+            display: block;
+        }
+    }
```
Comment (Igloczek) [inline components/02-elements/list/_list.scss]: @talalus  You mixed up here a `horizontal` with `divided` + custom code, that's not correct aproach. Remember that components needs to be reusable and highli customizable, not created just to use it in one place.
I already have rewritten this component in develop branch, so just revert these changes and adjust your code in the theme.

Comment (Igloczek) [inline components/02-elements/list/_list.scss]: I already merged develop here, b/c of conflicts, so just update your code in the theme, to use new modificators.

Comment (talalus) [inline components/02-elements/list/_list.scss]: done

File: components/02-elements/list/list.config.js [hunk 2]
```diff
@@ -66,6 +66,18 @@ module.exports = {
...
     {
+      name: 'center',
+      context: {
+        modificators: 'list--center'
+      }
+    },
+    {
+      name: 'center-horizontal',
+      context: {
+        modificators: 'list--center list--horizontal'
```
Comment (Igloczek) [inline components/02-elements/list/list.config.js]: You are defining `modificators` in config but using `class` in template. Rename everyting to `class`.

File: components/03-modules/form/_form.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        .input__label {
+            margin: 0 10px 0 0;
+        }
+
+        .button {
+            margin: 0 0 0 (-$button-icon-border-width);
+        }
+    }
+
+    div.mage-error[generated] {
```
Comment (Igloczek) [inline components/03-modules/form/_form.scss]: This one should stay in theme, it's heavily Magento related, not useful in components at all.

File: components/03-modules/form/_form.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        .input {
+            margin-bottom: 0;
+            width: 100%;
+            display: flex;
+            flex-wrap: nowrap;
+            align-items: center;
+            justify-content: center;
+        }
+
+        .input__field {
```
Comment (Igloczek) [inline components/03-modules/form/_form.scss]: `&__field`

File: components/03-modules/form/_form.scss (hunk 3 above)
Comment (talalus) [inline components/03-modules/form/_form.scss]: I moved it to component because it's necessary to be here

File: components/02-elements/list/list.config.js (hunk 2 above)
Comment (talalus) [inline components/02-elements/list/list.config.js]: renamed to class

File: components/03-modules/form/_form.scss (hunk 4 above)
Comment (talalus) [inline components/03-modules/form/_form.scss]: fixed

File: components/03-modules/form/_form.scss (hunk 3 above)
Comment (Igloczek) [inline components/03-modules/form/_form.scss]: No, is not. It's only related to Magento, so should be added to theme (I already wrote the same thing in theme https://i.imgur.com/gl4OeWn.png)

File: components/02-elements/button/button.config.js [hunk 5]
```diff
@@ -21,15 +21,6 @@ module.exports = {
...
       }
-    },
-    {
-      name: 'link',
-      context: {
-        tag: 'a',
-        class: 'button--link',
-        attributes: 'href="#"',
-        text: 'I am a link button',
-      }
```
Comment (Igloczek) [inline components/02-elements/button/button.config.js]: Any idea why it's removed?

File: components/03-modules/form/_form.scss (hunk 3 above)
Comment (talalus) [inline components/03-modules/form/_form.scss]: done

File: components/02-elements/button/button.config.js (hunk 5 above)
Comment (talalus) [inline components/02-elements/button/button.config.js]: by mistake during the merge, fixed now
